exasol_integration_test_docker_environment/lib/data/database_info.py

```diff
@@ -1,13 +1,12 @@
 from exasol_integration_test_docker_environment.lib.base.info import Info
 from exasol_integration_test_docker_environment.lib.data.container_info import ContainerInfo
+from exasol_integration_test_docker_environment.lib.test_environment.ports import PortForwarding
 
 
 class DatabaseInfo(Info):
-
-    def __init__(self, host: str, db_port: str, bucketfs_port: str,
+    def __init__(self, host: str, ports: PortForwarding,
```

Comment:
This change forces also a major release, because DatabaseInfo is used in return values of the API

Reply:
Actually it would have been much easier to add the SSH port forwarding if a class like PortForwarding would have been used before 🙂 
On the other hand currently I do not expect another port to be forwarded, soon. 

Which way should we go?
* A) encapsulate the ports into a class which I find very useful, easier to maintain and also shortens and simplifies the code and gains very readable names in my eyes.
* B) keep backwards compatibility (for which I cannot rate the importance)

Reply:
A and go to version 2.0.0, we should have mostly internal users

Reply:
Thanks. I will do so.